models/lido/liquidity/arbitrum/lido_liquidity_arbitrum_curve_pools.sql

```diff
@@ -102,40 +107,40 @@ select explode(sequence(to_date('{{ project_start_date }}'), now(), interval 1 d
     from (
     select date_trunc('day', evt_block_time) as time
         , contract_address as pool
-        , 0 as eth_amount_raw
+        , cast(0 as uint256) as eth_amount_raw
```

Comment:
only going to comment on this once for simplicity, but if you could apply changes as requested in #3760 throughout this PR too?